### PR TITLE
Simplify redecs_agent to run inside a container

### PR DESCRIPTION
**Pull request by mishan**
It will expect SERVICE_NAME to be defined in the environment

In retrospect, there wasn't really a good reason to run it on the EC2 instance.

### CODE REVIEW

**Comment by mishan** [comment]
@TysonAndre

File: redecs_agent/redecs_agent.go [hunk 1]
```diff
@@ -60,58 +55,9 @@ func reportServiceActive(serviceName string) error {
...
-		}
-	}
-
-	log.Debug("Done checking active containers")
+// Gets the service name from the SERVICE_NAME environment variable.
+func getServiceName() string {
+	return os.Getenv("SERVICE_NAME")
 }
 
 func main() {
```
Comment (TysonAndre) [inline redecs_agent/redecs_agent.go:63]: `err = errors.New(fmt.Sprintf("Usage: %s [Redis host] {host IP (for testing)}\n", os.Args[0]))`

should say current IP instead?

File: redecs_agent/redecs_agent.go [hunk 2]
```diff
@@ -158,64 +99,14 @@ func main() {
...
-	logErrorAndFail(err)
-	dockerRouter.Start()
-
-	log.Debug("Waiting for Docker events")
+	log.Debug("redecs_agent started")
 
 	// continue processing once per minute
-	ticker := time.NewTicker(checkInterval)
+	ticker := time.NewTicker(reportInterval)
+	serviceName := getServiceName()
```
Comment (TysonAndre) [inline redecs_agent/redecs_agent.go]: Load this at the very beginning, exit with a descriptive error message and non-zero exit code if serviceName the empty string (https://godoc.org/os#Getenv)

- This script is misconfigured (or being used improperly) if there is no service name

**Review by TysonAndre — APPROVED** [review]
Seems fine overall, some minor comments

File: redecs_agent/redecs_agent.go (hunk 1 above)
Comment (mishan) [reply redecs_agent/redecs_agent.go:63]: Derp, yes, forgot to update that, thanks

File: redecs_agent/redecs_agent.go (hunk 2 above)
Comment (mishan) [reply redecs_agent/redecs_agent.go]: Good point! Will fix.

**Pull request merged**